.circleci/config.yml

```diff
@@ -18,15 +18,15 @@ jobs:
       - run:
           name: Checking formating
           command: sbt scalafmtCheckAll
-#      - run: TODO: uncomment when #24 is done
-#          name: Checking import order
-#          command: sbt "scalafixAll --check"
+      - run:
+          name: Checking import order
+          command: sbt "scalafixAll --check"
       - run:
           name: Compiling production code
           command: sbt compile
       - run:
           name: Compiling tests
-          command: sbt test:compile
+          command: sbt Test/compile
```

Comment:
sbt CLI/REPL is madness 🤯 I've got no idea why this was needed but I trust you it was 😅